doc/security_need_to_know.md

```diff
@@ -0,0 +1,28 @@
+# About security issues
+
+This document aims to explain the process of identifying a security issue and the steps for managing a security issue.
```

Comment:
the expected audiences and purposes of this document are not obvious to me.

Reply:
The document is intended to:
- Clarify to the community and TSC which types of issues will be recognized as security issues.
- Provide a step-by-step guide on how to handle a security issue once it has been identified.
- Ensure maintainers distinguish security issues from all reported community issues and address them with caution.
- Encourage accurate reporting of security issues, rather than categorizing them as normal issues.